### PR TITLE
added send action to whatsapp

### PR DESCRIPTION
**Pull request by Dreamersoul**

### CODE REVIEW

File: Appz/Appz/Apps/WhatsApp.swift [hunk 1]
```diff
@@ -41,6 +42,15 @@ extension Applications.WhatsApp.Action: ExternalApplicationAction {
                 ),
                 web: Path()
             )
+        case .Send(let abid, let text):
+            return ActionPaths(
+                app: Path(
+                    pathComponents: ["send"],
+                    queryParameters: ["text" : text, "abid": abid ?? ""]
```
Comment (Mazyod) [inline Appz/Appz/Apps/WhatsApp.swift:49]: You don't need to use null coalescing `??` here, unless `abid` is an optional string `String?`.

Comment (Dreamersoul) [inline Appz/Appz/Apps/WhatsApp.swift:49]: Fixed and pushed

**Comment by Dreamersoul** [comment]
oh shoot, sorry pushed into the original repo :/

**Comment by Mazyod** [comment]
lol, I was wondering why you merged it so fast, before even Travis returned :white_check_mark: 

No worries, it's a great contribution, with just a minor nitpick :eyeglasses:

**Pull request merged**